apps/sms/test/marionette/lib/messages.js

```diff
@@ -63,11 +64,11 @@ var ParticipantsAccessor = require('./views/participants/accessors');
 
         Participants: new ParticipantsAccessor(client),
 
-        launch: function() {
+        launch() {
           client.switchToFrame();
           client.apps.launch(ORIGIN_URL);
           client.apps.switchToApp(ORIGIN_URL);
-          client.helper.waitForElement(SELECTORS.main);
+          client.helper.waitForElement(SELECTORS.appReady);
```

Comment:
Seems like we can remove SELECTORS.main because we didn't need it for other place?